src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2315,6 +2303,11 @@ void ApplyPreserveInfo (TypeDefinition type)
 				var di = new DependencyInfo (DependencyKind.TypePreserve, type);
 
 				switch (members) {
+				case TypePreserveMembers.All:
```

Comment:
they are enum values stored into a different annotation

Reply:
I can see that they're different enums. What I mean is, we have this case above:
```csharp
case TypePreserve.All:
	MarkFields (type, true, di);
	MarkMethods (type, di, type);
	return;
```

Why do we need `TypePreserveMembers.All` if it is only duplicating this logic? Could we do `Annotations.SetPreserve (type, TypePreserve.All)` below, instead of `Annotations.SetMembersPreserve (type, TypePreserveMembers.All)`? That seems to be the only place we use it.